sql-cli/CHANGELOG.md

```diff
@@ -1,5 +1,39 @@
 # Changelog
 
+## 0.2.0
+
+A feature release containing the following major features:
+
+* load without Python
+* dynamic DAG generation
+* improve initialisation time or give visibility that it will take longer than expected
```

Comment:
```suggestion
* improve project initialisation time or give visibility that it will take longer than expected
```

Reply:
This is actually not true. It is referring to the init time docker needs to setup (build & run) the env.

Reply:
```suggestion
```
Maybe in that case we should not mention this here at all.